src/appMain/sdl_preloaded_pt.json

```diff
@@ -3721,7 +3733,210 @@
                             ],
                             "key": "OEM_REF_IN_RIGHT_REAR",
                             "type": "Struct",
-                            "mandatory": true
+                            "mandatory": true,
+                            "since": "2.0",
+                            "untill": "7.1"
+                        }
+                    ],
+                    "key": "OEM_REF_TIRE_PRES",
+                    "type": "Struct",
+                    "mandatory": false
+                },
+                {
+                    "name": "tirePressure",
```

Comment:
We don't need a new vehicle data item, this way it'd just be ignored. You should rather create a duplicate sub-param next to current sub-param. Both would be differentiated by `since` / `until` fields. 

See example for `utcYear` from [here](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/sdl_preloaded_pt.json#L3076) till [here](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/sdl_preloaded_pt.json#L3093)

Reply:
@atiwari9 Thank you. Please check the updates in 3cee9a1a69489b1c38a04050e64d397d6ddbef09